connect/runtime/src/test/java/org/apache/kafka/connect/integration/ExactlyOnceSourceIntegrationTest.java

```diff
@@ -768,27 +767,32 @@ public void testSeparateOffsetsTopic() throws Exception {
                     recordNum >= recordsProduced);
```

Comment:
I know this is out of the scope of this fix, but I noticed that the `recordNum >= recordsProduced` seems to conflict with the `// consume all` part of the comment. Or perhaps the call to `consume` on line 760 should be `consumeAll` too?

Reply:
Ahh, good catch--the "all" is incorrect in that comment. I've updated it from "consume all records" to "consume at least the expected number of records".

On a separate note, I believe a similar mixup is the cause of https://issues.apache.org/jira/browse/KAFKA-14101, where we _should_ be using `consumeAll` but aren't at the moment. I've pushed a commit that should fix that.